### ISSUE TITLE
FILE_NAME variable across several controls

### ISSUE THREAD

**Issue by ejaronne** [djhaynes/inspec-profile-disa_stig-el7#39]
Possible inaccurate testing and visible results display:

Controls [V-72175](https://github.com/djhaynes/inspec-profile-disa_stig-el7/blob/master/controls/V-72175.rb), [V-73165](https://github.com/djhaynes/inspec-profile-disa_stig-el7/blob/master/controls/V-73165.rb), [V-73167](https://github.com/djhaynes/inspec-profile-disa_stig-el7/blob/master/controls/V-73167.rb), [V-73171,](https://github.com/djhaynes/inspec-profile-disa_stig-el7/blob/master/controls/V-73171.rb) [V-73173](https://github.com/djhaynes/inspec-profile-disa_stig-el7/blob/master/controls/V-73173.rb) use FILE_NAME constant, giving inconsistent results display and may affect actual testing. A test might report 
"SKIP -- The /etc/gshadow file does not exist, this requirement is Not Applicable."
when in fact it was supposed to be testing /etc/opassword

**Comment by samcornwell** [comment]
This is being fixed by simp/inspec-profile-disa_stig-el7#20. Once that is merged into simp, we can pull it back here.

**Comment by samcornwell** [comment]
Fixed by a48520f535136588c87b511ae16ab5034234ee18

**Comment by samcornwell** [comment]
Note, this is just in https://github.com/djhaynes/inspec-profile-disa_stig-el7/tree/dev for now
